calico-enterprise/observability/elastic/archive-storage.mdx

```diff
@@ -206,3 +209,6 @@ $[prodname] uses Splunk's **HTTP Event Collector** to send data to Splunk server
 </TabItem>
 
 </Tabs>
+
+### Configure log archival
```

Comment:
I think you can have an additional task section here. But first, a more descriptive title: "Configure log archiving for non-cluster hosts or VMs" or something like that.